### PR TITLE
Add high contrast styles for persona presence

### PR DESCRIPTION
**Pull request by philworthington**
Adds high contrast styles to match Skype in high contrast mode. Updates presence icon size to 8px for proper alignment.

### CODE REVIEW

File: src/components/Persona/Persona.scss [hunk 1]
```diff
@@ -202,11 +202,25 @@ $ms-Persona-presenceSizeXl: 28px;
   bottom: -1px;
   border: 2px solid $ms-color-white;
   text-align: center;
+
+    @media screen and (-ms-high-contrast: active) {
```
Comment (Jahnp) [inline src/components/Persona/Persona.scss]: nit: indentation here looks funny--I'd double-check before closing

**Comment by Jahnp** [comment]
Approved :yellow_heart:

File: src/components/Persona/Persona.scss (hunk 1 above)
Comment (philworthington) [inline src/components/Persona/Persona.scss]: Looks like it was spaced one too many times. Fixed

**Comment by mikewheaton** [comment]
Approved 🌵

**Pull request merged**